### PR TITLE
React BookStore Setup

### PR DESCRIPTION
**Pull request by paulshaun-kdtk**
# CHANGES

- Added form for book submittion
- styled book shelf card

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+// end imports
+
+function App() {
+  return (
+    <Router>
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/books" element={<Books />} />
+        <Route path="/components" element={<Components />} />
+      </Routes>
```
Comment (fobadara) [inline src/App.js]: [REQUIRED] You should have routes to home and categories page. But I can't see any route to categories page. Please fix this.

File: src/components/Books.js [hunk 2]
```diff
@@ -0,0 +1,163 @@
+import React, { useState } from 'react';
+import './Main.css';
+import './Book.css';
+import { CircularProgressbar, buildStyles } from 'react-circular-progressbar';
+import Header from './Header';
+import 'react-circular-progressbar/dist/styles.css';
+
+function Book({
```
Comment (fobadara) [inline src/components/Books.js:8]: [OPTIONAL] I think you should move this to another file. This helps readability and ensures separation of concerns.

**Review by fobadara — APPROVED** [review]
Hi @paulshaun-kdtk, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

REDO

## Highlights
✅ Descriptive README
✅ Descriptive PR summary
✅ Clean code
✅ Good first attempt

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @paulshaun-kdtk 

Your project is complete! Great job 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- routes well set 
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**